src/black/lines.py

```diff
@@ -484,10 +484,10 @@ def __str__(self) -> str:
         leaves = iter(self.leaves)
         first = next(leaves)
         res = f"{first.prefix}{indent}{first.value}"
-        for leaf in leaves:
-            res += str(leaf)
-        for comment in itertools.chain.from_iterable(self.comments.values()):
-            res += str(comment)
+        res += ''.join(str(leaf) for leaf in leaves)
+        comments_iter = itertools.chain.from_iterable(self.comments.values())
+        comments = [str(comment) for comment in comments_iter]
+        res += ''.join(comments)
```

Comment:
```suggestion
        res += "".join(str(leaf) for leaf in leaves)
        comments_iter = itertools.chain.from_iterable(self.comments.values())
        comments = [str(comment) for comment in comments_iter]
        res += "".join(comments)
```
The Black style uses double quotes, which is the main source of CI failing
(there's also a longstanding issue #4674 but that can be ignored)